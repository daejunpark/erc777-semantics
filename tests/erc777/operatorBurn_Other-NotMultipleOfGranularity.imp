test.preamble();
test.setGranularity(10);

// Testing operatorBurn(From, Value, OperatorData) when From =/= caller and Value is not multiple of granularity().
test.makeCaller(8);
test.operatorBurn(7, 23, "");
