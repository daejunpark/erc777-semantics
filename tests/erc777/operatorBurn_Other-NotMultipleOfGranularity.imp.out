granularity: 10
Changing the caller to 8
8 is burning 23 units from 7 (operatorData:"")
REVERT
Balance of 7 is 70
Total supply is 550
